stable_baselines/common/policies.py

```diff
@@ -8,6 +8,8 @@
 from stable_baselines.common.distributions import make_proba_dist_type
 from stable_baselines.common.input import observation_input
 
+import warnings
```

Comment:
Minor: please put built-in import first.
@hill-a we should write those rules somewhere, so for imports order:
1. built-in
2. packages
3. current module

with one space between each,  that gives for instance:
```python
import os
import warnings

import numpy as np

from stable_baselines import PPO2
```